app/helpers/SessionAnswersHelper.scala

```diff
@@ -96,21 +96,21 @@ class SessionAnswersHelper @Inject()(uploadJourneyRepository: UploadJourneyRepos
 
       case "crime" => Seq(
         (messages("checkYourAnswers.reasonableExcuse"),
-          messages(s"checkYourAnswers.${request.session.get(SessionKeys.reasonableExcuse).get}.reasonableExcuse"),
+          messages(s"reasonableExcuses.${request.session.get(SessionKeys.reasonableExcuse).get}Reason"),
           changeAnswerUrl(
             controllers.routes.ReasonableExcuseController.onPageLoad().url,
             ReasonableExcuseSelectionPage
           )
         ),
-        (messages("checkYourAnswers.crime.whenDidTheCrimeHappen"),
+        (messages("crimeReason.headingAndTitle"),
           dateToString(LocalDate.parse(request.session.get(SessionKeys.dateOfCrime).get)),
           changeAnswerUrl(
             controllers.routes.CrimeReasonController.onPageLoadForWhenCrimeHappened(CheckMode).url,
             WhenDidCrimeHappenPage
           )
         ),
-        (messages("checkYourAnswers.crime.hasCrimeBeenReported"),
-          messages(s"checkYourAnswers.crime.${request.session.get(SessionKeys.hasCrimeBeenReportedToPolice).get}"),
+        (messages("crimeReason.hasBeenReported.headingAndTitle"),
+          messages(s"common.radioOption.${request.session.get(SessionKeys.hasCrimeBeenReportedToPolice).get}"),
```

Comment:
You may have to add a common.radioOption.unknown message key, as the answer to store is unknown for has crime been reported, see the attached:
![image](https://user-images.githubusercontent.com/53828896/161015318-50a139f0-ca34-4217-aee3-c368f50596d0.png)